xc/xc7/tests/install_test/CMakeLists.txt

```diff
@@ -2,11 +2,6 @@ set(INSTALLATION_DIR_BIN "${CMAKE_INSTALL_PREFIX}/bin")
 
 add_test(NAME binary_toolchain_test
 	COMMAND ${CMAKE_COMMAND} -E env
-	PATH=${INSTALLATION_DIR_BIN}:$ENV{PATH}
-	${CMAKE_COMMAND} -E env
-	PYTHONPATH=${PRJXRAY_DIR}:${PRJXRAY_DIR}/third_party/fasm
-	XRAY_FASM2FRAMES=${PRJXRAY_DIR}/utils/fasm2frames.py
-	XRAY_TOOLS_DIR=${symbiflow-arch-defs_SOURCE_DIR}/build/third_party/prjxray/tools
-	XRAY_DATABASE_DIR=${PRJXRAY_DB_DIR}
-	make
+		PATH=${INSTALLATION_DIR_BIN}:$ENV{PATH}
+		make
```

Comment:
I believe this change could be a rebase error. I guess the `Install CI` failure is due to this.

Reply:
Ya